pkg/webhookconfig/registration.go

```diff
@@ -151,14 +174,38 @@ func (wrc *Register) Remove(cleanUp chan<- struct{}) {
 	wrc.removeSecrets()
 }
 
+func (wrc *Register) getResourceWebhookRules() []interface{} {
+	kinds := wrc.policyCache.GetKinds()
+	logger := wrc.log.WithName("getResourceWebhookRules")
+	result := make([]interface{}, 0, len(kinds))
+	for _, kind := range kinds {
+		_, gvr, err := wrc.client.DiscoveryClient.FindResource("", kind)
```

Comment:
Does apiversion being hardcoded as empty string cause any issues with non core resources? I haven't looked to deeply

Reply:
i haven't tested it with non core resources actually / but yeah potentially an issue

Reply:
i believe i didn't write this code ^^^ but same diff i will fix it if there is an issue / soon as we create a policy for a CRD